### DISCUSSION TITLE
Help with the vulnerability detector

### DISCUSSION THREAD

**Discussion by styxiner** [wazuh/wazuh#24189] [Q&A] [answered]
So I have a wazuh server on a oracle 9 server and I don't see on the dashboard any vulnerability
![image](https://github.com/wazuh/wazuh/assets/160549592/eaec3b9d-cca6-4282-baa2-46790ce248d0)

The ossec.conf on the server for that is the following:

```
<vulnerability-detection>
    <enabled>yes</enabled>
    <index-status>yes</index-status>
    <feed-update-interval>60m</feed-update-interval>
  </vulnerability-detection>

  <indexer>
    <enabled>yes</enabled>
    <hosts>
      <host>https://wazuh.indexer:9200</host>
    </hosts>
    <ssl>
      <certificate_authorities>
        <ca>/etc/ssl/root-ca.pem</ca>
      </certificate_authorities>
      <certificate>/etc/ssl/filebeat.pem</certificate>
      <key>/etc/ssl/filebeat.key</key>
    </ssl>
  </indexer>
```

**Comment by sebasfalcone** [comment]
Hi @Jaime-Moranchel sorry for the delay

I will recommend following the [troubleshooting guide](https://documentation.wazuh.com/current/upgrade-guide/troubleshooting.html)

---

Most of the time, this issue is related to the manager not being able to connect to the indexer. We could verify the certificates first:
- `filebeat test output`
- `curl -u USER:PASS --cacert /etc/ssl/root-ca.pem --cert /etc/ssl/filebeat.pem --key /etc/ssl/filebeat.key -X GET "https://wazuh.indexer:9200/_cat/indices/*vuln*?v"`

> [!NOTE]
> Replate USER and PASS values with the corresponding ones

**Comment by styxiner** [comment, marked as answer]
I close the issue because I endeda up reinstalling it and worker months ago
